ci/docker/android-emulator/prepare_snapshot.sh

```diff
@@ -103,3 +103,5 @@ adb emu kill
 sleep 5
 
 adb kill-server
+
+echo "Emulator preparation finished"
```

Comment:
Final message for automation and debugging.
Previously, we relied only on exit code and timeout.